src/open_inwoner/kvk/client.py

```diff
@@ -122,6 +125,24 @@ def get_all_company_branches(self, kvk: str, **kwargs) -> list[dict | None]:
 
         return branches
 
+    @cache("vestigingsnummer")
```

Comment:
That's not a use case because the only `kwarg` passed on is the `vestigingsnummer`. The code is misleading though; `kwargs.update` suggests that there is other stuff in there. I changed the function signature + body to make clear that we're only passing on the `vestigingsnummer`.